specification/trace/semantic_conventions/cloudevents.md

```diff
@@ -13,8 +18,33 @@ The attributes described in this document are not specific to a particular opera
 
 <!-- tocstop -->
 
+## Spans
+
+### Creation
+
+Instrumentation SHOULD create a new span and populate 'Distributed Context Extension' on the event. It applies when:
+
+- CloudEvent is created by instrumented library
```

Comment:
Based on our conversation earlier, I changed it to 
> CloudEvent is created by instrumented library. It may be impossible or impractical to create span in CloudEvent constructor (or factory) - instrumentation MAY create span later when passing event to transport layer.

LMK if it helps